src/storage/log/mod.rs

```diff
@@ -10,11 +10,8 @@ use std::io::{self, BufRead, Read, Seek, SeekFrom, Write};
 use std::path::{Path, PathBuf};
 use std::sync::PoisonError;
 
-/// The size of a single block in bytes.
-///
-/// The `BLOCK_SIZE` constant represents the size of a block used for buffering disk writes in the
-/// write-ahead log. It determines the maximum amount of data that can be held in memory before
-/// flushing to disk. Larger block sizes can improve write performance but might use more memory.
+/// The `BLOCK_SIZE` constant represents the size of a block used for buffering disk reads from
+/// the append-only log.
 pub(crate) const BLOCK_SIZE: usize = 32 * 1024;
```

Comment:
We won't need this constant now right?

Reply:
For this one we still need this. I've got another PR waiting for this to get merged, where I will remove it.